_data/rules.yml

```diff
@@ -270,6 +270,10 @@ services:
       - name: danielqsj/kafka_exporter
         doc_url: https://github.com/danielqsj/kafka_exporter
         rules:
+          - name: Kafka Topics 
+            description: Kafka topic in-sync partition
+            query: 'sum(kafka_topic_partition_in_sync_replica) by (topic) > 3'
```

Comment:
This should trigger an alert for `> 3` or `< 3` ?

Reply:
mmm should be < 3, my fault let me fix that.

The in-sync replicas should accomplish the minimum set, if it is less than the minimum 3, then the Producers raises an Exception.